persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -1390,6 +1397,9 @@ takeForeign ps entityName = takeRefTable
                                 EntityNameHS refTableName
                             , foreignRefTableDBName =
                                 EntityNameDB $ psToDBName ps refTableName
+                            , foreignRefSchemaDBName =
+                                Nothing
+                            -- ^ This will be determined in the TH phase ('fixForeignRefSchemaDBName').
```

Comment:
Even though this is written as a haddock comment, because it's in actual implementation code, people will have to read the internal module to see it.

If this is information about the lifecycle contract of the values on the record type, it might be worth hoisting this to something like the type definition so it's apparent in docs